### PR TITLE
Doc for publishing to IIS with MSDeploy

### PR DESCRIPTION
**Pull request by robbert229**
From issue #766. 

@sayedihashimi Is everything good?

Failing builds are yet again not from anything I have done.... so confused.

### CODE REVIEW

File: aspnet/publishing/iis-with-msdeploy.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
+Publishing to IIS with Web Deploy using Visual Studio 2015
+============================================================
+
+By `Sayed Hashimi`_
```
Comment (sayedihashimi) [inline aspnet/publishing/iis-with-msdeploy.rst]: Can you change to Sayed Ibrahim Hashimi? My brothers all have the same name and we all love taking credit for each others work :)

File: aspnet/publishing/iis-with-msdeploy.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+- `64 bit HttpPlatformHandler <http://go.microsoft.com/fwlink/?LinkID=690721>`_
+- `32 bit HttpPlatformHandler <http://go.microsoft.com/fwlink/?LinkId=690722>`_
+
+In addition to installing the HttpPlatformHandler, you’ll need to install the
+latest version of Web Deploy (version 3.6). To install Web Deploy 3.6 you can
+use the `the Web Platform Installer
+<https://www.microsoft.com/web/downloads/platform.aspx>`_. (WebPI) or `directly
+from the download center
+<https://www.microsoft.com/en-us/download/details.aspx?id=43717>`_. The
+preferred method is to use the WebPI. WebPI offers a standalone setup as well as
```
Comment (sayedihashimi) [inline aspnet/publishing/iis-with-msdeploy.rst]: "to use WebPI"

File: aspnet/publishing/iis-with-msdeploy.rst [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+you’re hosting provider has support for creating a publish profile, download
+that and then import it into the Visual Studio publish dialog with the Import
+button. You can see that dialog shown below.
+
+.. image:: iis-with-msdeploy/_static/pub-dialog.png
+
+After importing the publish profile, there is one additional step that needs to
+be taken before being able to publish to a standard IIS host. In the publish
+PowerShell script generated (under Properties\PublishProfiles) update the
+publish module version number from 1.0.1 to 1.0.2-beta2. After changing 1.0.1 to
```
Comment (sayedihashimi) [inline aspnet/publishing/iis-with-msdeploy.rst]: Can we format version numbers as code?

**Comment by robbert229** [comment]
@sayedihashimi The requested changes have been implemented.

**Comment by sayedihashimi** [comment]
@robbert229 thanks Robert.

@danroth27 what's the next step in getting this PR merged?

**Comment by danroth27** [comment]
@rachelappel Please merge.

**Comment by rachelappel** [comment]
Merged. Thanks for contributing! bab1447b315c3cc734830555a0faa4442e604fcf

**Comment by sayedihashimi** [comment]
Thanks everyone

**Comment by GuardRex** [comment]
@Rick-Anderson Probably should have the link to this in **Publishing to IIS** at this location:

> Note: MSDeploy is the recommended mechanism for deployment, but you can use Xcopy, Robocopy or another approach.

https://docs.asp.net/en/latest/publishing/iis.html?highlight=msdeploy#deploy-to-iis-server

**Pull request closed without merging**